sass/layout/_sidebar-left.scss

```diff
@@ -257,6 +257,82 @@
         outline-style: none;
     }
 
+    .SidebarMenu {
+        height: 30px;
+        opacity: 1;
+
+        .dropdown-menu {
+            line-height: 20px;
+            min-width: 0;
+            padding: 8px 0;
+
+            span {
+                white-space: nowrap;
+            }
+
+            button {
+                padding: 1px 16px;
+            }
+
+            i {
+                font-size: 16px;
+                color: var(--center-channel-color-72);
+                line-height: 16px;
+            }
+        }
+
+        .Menu {
+            position: fixed;
```

Comment:
can we consider doing more BEM?

Reply:
Sure, again for the `Menu` stuff we'll address that outside of this PR.